src/pool/pool_jemalloc.c

```diff
@@ -461,7 +470,9 @@ static umf_result_t op_initialize(umf_memory_provider_handle_t provider,
         err = je_mallctl("arenas.create", (void *)&arena_index, &unsigned_size,
                          NULL, 0);
         if (err) {
-            LOG_ERR("Could not create arena.");
+            LOG_ERR("Could not create a jemalloc arena (n_arenas = %zu, i = "
```

Comment:
nit: another solution to consider would be to start with utils_get_num_cores() * 4 and then try lower numbers in a loop until we got no error. I'm not sure if we need such a complicated solution, but what if there would be 1000 cores machine in the future?

Reply:
Done